### PR TITLE
added parameter apply_to for rabbitmq policy

### PR DESCRIPTION
**Pull request by dnABic**
Hello,

### What does this PR do?

Added parameter 'apply_to' for method called 'set_policy' in rabbitmq module. Backwards compatibility is preserved. 

### What issues does this PR fix or reference?
For RabbitMQ, an policy can be applied to queues, exchanges or both of those entities. So far, saltstack rabbitmq module was not using this parameter. Therefore, by default all policies was applied to both, rabbitmq queues and exchanges. This is not always expected behavior.

Saltstack is using rabbitmqctl command line tool to create rabbitmq policies. More about parameter '--apply-to' can be found here: https://www.rabbitmq.com/man/rabbitmqctl.1.man.html

### Previous Behavior
Created rabbitmq policy was always applied to both, queues and exchanges.

### New Behavior
Possibility to select if rabbitmq policy is applied to queues, exchanges or both.

### Tests written?

If requested, I will write test. There is already an test for set_policy and it is expected to work with changes in PR.

Also, I have tested my code changes on Rabbitmq with saltstack formula: https://github.com/saltstack-formulas/rabbitmq-formula
with the following pillar:
https://gist.github.com/dnABic/0c0ab8fd882444fe3239c69dd5749196

### CODE REVIEW

File: salt/modules/rabbitmq.py [hunk 1]
```diff
@@ -848,7 +848,7 @@ def list_policies(vhost="/", runas=None):
     return ret
 
 
-def set_policy(vhost, name, pattern, definition, priority=None, runas=None):
+def set_policy(vhost, name, pattern, definition, apply_to=None, priority=None, runas=None):
     '''
```
Comment (oz123) [inline salt/modules/rabbitmq.py]: @dnABic 
For backwards compatibility, I would suggest that `apply_to` would follow `priority`.

**Comment by rallytime** [comment]
Go Go Jenkins!

File: salt/modules/rabbitmq.py (hunk 1 above)
Comment (cachedout) [inline salt/modules/rabbitmq.py]: @dnABic What do you think about making the change that @oz123 has suggested here?

**Comment by cachedout** [comment]
Go Go Jenkins!

**Review by rallytime — APPROVED** [review]

**Comment by rallytime** [comment]
Thanks for submitting this @dnABic and welcome!

**Pull request merged**